utils/pre-publish-check.js

```diff
@@ -3,8 +3,8 @@
  * Pre-publish checks to verify that our publish will go smoothly.
  */
 const path = require("path");
-const glob = require("glob");
 const {exec} = require("child_process");
+const glob = require("glob");
```

Comment:
Oooo, we should update this to fast-glob.

No action required.

Reply:
I'll update it separately.  It feels like we should almost have a linter for this, lol.